vignettes/relase-process.qmd

```diff
@@ -0,0 +1,155 @@
+---
+title: "The Release Process"
+description: >
+  Reference for how to perform R package releases in the Hubverse
+vignette: >
+  %  \VignetteIndexEntry{The Release Process}
+  %  \VignetteEngine{quarto::html}
+  %  \VignetteEncoding{UTF-8}
+knitr:
+  opts_chunk: 
+    collapse: true
+    comment: '#>'
+---
+
+This document is adapted for The Hubverse from [The Carpentries Hubverse Developer's Handbook](https://carpentries.github.io/workbench-dev/releases.html) (c) The Carpentries. 
+
+## Checklist
+
+ - [ ] Create a new branch from `main` called `release-X.Y.Z`
+ - [ ] Update version number in DESCRIPTION and check that the `Remotes:` is
+       up-to-date
+ - [ ] Update NEWS for the changes in this version
+ - [ ] commit, push changes, and create pull request
+ - [ ] get a review from another member of the Hubverse dev team and merge on approval
+ - [ ] add new tag (annotated (`-a`) or signed (`-s`) preferred) with the name "<package> X.Y.Z" and push
+      
+      ```bash
+      # example: create a signed (-s) tag for version 3.3.3
+      git tag -s 3.3.3 -m '<short explanation of what changed>'
+      git push --tags
+      ```
+ - [ ] create a release on github from the new tag
+
```

Comment:
I would also appreciate a section here on how we manage testing interdependent packages, both the stable and development versions.

Reply:
That's a good point. I think I'm going to make a separate vignette. For now, I've created a repository that tests the releases for our universe weekly across windows, mac, and ubuntu (oldrel, release, and devel): https://github.com/zkamvar/universe-tester

Reply:
I've added a section at the bottom